client/asset/dcr/dcr.go

```diff
@@ -393,6 +394,10 @@ type combinedClient struct {
 	*walletClient
 }
 
+func (cc *combinedClient) ValidateAddress(ctx context.Context, address dcrutil.Address) (*walletjson.ValidateAddressWalletResult, error) {
```

Comment:
Maybe a quick doc string explaining why this is necessary (disambiguate the node and wallet methods).